packages/truffle/test/scenarios/commands/console.js

```diff
@@ -1,23 +1,32 @@
 const CommandRunner = require("../commandRunner");
 const Server = require("../server");
-const Config = require("@truffle/config");
 const MemoryLogger = require("../MemoryLogger");
 const assert = require("assert");
+const path = require("path");
+const Reporter = require("../reporter");
+const sandbox = require("../sandbox");
 
 describe("truffle console", () => {
-  let config;
+  let config, project;
   const logger = new MemoryLogger();
 
   before(async () => await Server.start());
   after(async () => await Server.stop());
 
-  before("before all setup", () => {
-    config = Config.default();
+  project = path.join(__dirname, "../../sources/console");
+
+  before(async function () {
+    this.timeout(10000);
+    config = await sandbox.create(project);
+    config.network = "development";
     config.logger = logger;
+    config.mocha = {
+      reporter: new Reporter(logger)
+    };
   });
 
-  describe("when run with --url option", () => {
-    it("displays the hostname in the prompt", async () => {
+  describe("when run with options", () => {
+    it("displays the url hostname in the prompt", async () => {
       const url = "http://localhost:8545";
       const parsedUrl = new URL(url);
```

Comment:
These tests should cover the cases
  - url
    - runs in folder with a truffle-config
    - runs in folder without a truffle-config
  - network
    - runs in folder with a truffle-config

Reply:
Sure, thanks for the suggestions @amal. I will add the tests to cover the mentioned cases.